### PR TITLE
Update recipe and license SPDX

### PR DESCRIPTION
**Pull request by fhoehle**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@fhoehle you should have commit rights to merge this one when it is ready.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,42 +1,40 @@
...
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/colour-{{ version }}.tar.gz
+  sha256: af20120fefd2afede8b001fbef2ea9da70ad7d49fafdb6489025dae8745c3aee
 
 build:
   noarch: python
-  number: {{ build }}
-  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (ocefpaf) [inline recipe/meta.yaml:20]: You may have to revert this one. I underestimated this package age!

Comment (fhoehle) [reply recipe/meta.yaml:20]: was not necessary, just down pinning `setuptools<=67` due to `d2to1` used for packaging. There is an upstream fix, but no reaction by maintainers.

**Comment by fhoehle** [comment]
this closes #4

**Pull request merged**